src/viem/preparedTransactionSerialization.ts

```diff
@@ -27,6 +31,15 @@ function mapBigIntsToStrings<T extends object, K extends keyof T>(
 }
 
 export type SerializableTransactionRequest = MapValuesToString<TransactionRequest, BigIntProps>
+type SerializablePreparedTransactionsPossible = Omit<
+  PreparedTransactionsPossible,
```

Comment:
yes you're right...i did a half hearted job here. i removed these types and started passing unserializable navigation params around, that resulted only in warnings but the app works. this didn't make me feel good, so i ended up making more explicit navigation parameters rather than passing the whole preparedTransactionsResult. i think i like where it ended up, what do you think?